### PR TITLE
Kole/css

### PR DESCRIPTION
**Pull request by katiayn**

### CODE REVIEW

File: pyconbalkan/core/static/css/components/event-sidebar.css [hunk 1]
```diff
@@ -8,21 +8,37 @@
     top: 0;
     left: 0;
     bottom: 0;
-    z-index: 2;
-    background-color: #fff;
     width: 400px;
-    padding: 30px 30px 30px 70px;
+    padding: 20px 40px 40px 40px;
+    background-image: url(/static/img/sidebar-bg.png);
```
Comment (ulgens) [inline pyconbalkan/core/static/css/components/event-sidebar.css]: Is it possible to use {% url %} from Django here?

Comment (kolebre) [reply pyconbalkan/core/static/css/components/event-sidebar.css]: Probably, I'm a bit rusty, last time I worked with Django templating was 3 years ago ;)

Comment (kolebre) [reply pyconbalkan/core/static/css/components/event-sidebar.css]: Well, it seems it's not possible, the static is only for templates 😄

File: pyconbalkan/conference/templates/home.html [hunk 2]
```diff
@@ -66,4 +66,4 @@ <h2 class="title title--white title--medium title--uppercase mt-xs-20">
 
     <a class="button button--yellow mb-xs-80" href="https://www.eventbrite.com/e/pycon-balkan-tickets-45876685401?ref=ebtnebtckt" target="_blank" role="button">Buy Tickets</a>
 
-{% endblock %}
+{% endblock %}
```
Comment (ulgens) [inline pyconbalkan/conference/templates/home.html]: Why are you deleting file endings? :smile:

**Comment by ulgens** [comment]
Guys, can we squash commit before merging this PR? That list is huge.

**Comment by katiayn** [comment]
@ulgens yes, sure! We can squash commit once it's done 😃✅

**Pull request merged**